configs/base/index.js

```diff
@@ -7,33 +7,6 @@ module.exports = {
   rules: {
     // Additions
     '@typescript-eslint/consistent-type-imports': ['error'],
-    '@typescript-eslint/naming-convention': [
-      'error',
-      {
-        selector: 'default',
-        format: ['strictCamelCase'],
-        leadingUnderscore: 'forbid',
-        trailingUnderscore: 'forbid',
-      },
-      {
-        selector: 'typeLike',
-        format: ['StrictPascalCase'],
-      },
-      {
-        selector: 'variable',
-        // Exception for FunctionComponents
-        format: ['strictCamelCase', 'StrictPascalCase', 'UPPER_CASE'],
```

Comment:
This one is not in the new config, only the version for `modifiers: ["exported"],` is.

So this isn't allowed anymore...
```ts
const MyComponent = defineComponent(...);
```

Which is often used in stories and tests – but not often in normal component files.

Should we still include this pattern in `.stories.ts` and `.test.ts` files?

Reply:
We can override the rule for `.stories.ts` and `.test.ts` files and allow `StrictPascalCase` for the `variable` selector only there. However, should we add this to [mediamonks/eslint-config](https://www.github.com/mediamonks/eslint-config) instead ?

Reply:
> However, should we add this to [mediamonks/eslint-config](https://www.github.com/mediamonks/eslint-config) instead?

Not sure, seems more Muban (and React) specific?

Reply:
agreed, I will add the override here and then in [mediamonks/eslint-config-react](https://github.com/mediamonks/eslint-config/tree/main/configs/react)